lobby_server.yaml

```diff
@@ -2,6 +2,9 @@
   host: lobby.triplea-game.org
   port: 3304
   message: |
+    IMPORTANT: The Lobby will be down for some maintenance on Thursday, the 30th November 2017 at 9:00am UTC.
+    Sorry for your inconvenience, maintenance shouldn't take longer than an hours.
```

Comment:
Might want to change "hours" to "hour" if your intention was to specify a one-hour window.

Reply:
Indeed, this happens, when I'm thinking about how long it'll take while writing it ^^
The last time I did some server upgrading things went horribly wrong (because PHP 5 was upgraded to PHP 7), and took me about 2 hours until the tripleawarclub server was up again...